clients/unshaded/src/main/java/tachyon/client/RemoteBlockReader.java

```diff
@@ -23,12 +23,14 @@
 import com.google.common.base.Throwables;
 
 import tachyon.Constants;
+import tachyon.annotation.PublicApi;
 import tachyon.conf.TachyonConf;
 import tachyon.util.CommonUtils;
 
 /**
  * The interface to read remote block from data server.
  */
+@PublicApi
```

Comment:
I think this is hidden from users who will only be using FileInStream/FileoutStream